app/views/partials/theme/simple/posts/card.liquid

```diff
@@ -14,45 +15,40 @@
       </a>
       {% include 'theme/simple/posts/vote_box', post: post, current_profile: current_profile, root_post: root_post %}
-    </div>
+    </aside>
     <div class="flex flex-col w-full h-full">
-      <div class="flex items-center justify-between">
-        <h2 class="text-lg font-semibold text-gray-900 -mt-1">
-          <a href="{{ creator_url }}">
+      <div class="md:flex items-start justify-between">
+        <header>
+          <a href="{{ creator_url }}" class="text-lg font-semibold -mt-1">
             {{post.creator.name}}
           </a>
-        </h2>
-        <small class="text-sm text-gray-700">{{post.created_at | l: 'compact'}}</small>
+          <date datetime="{{ post.created_at }}" class="block text-sm text-gray-700">{{post.created_at | l: 'compact'}}</date>
+        </header>
+        {% if post.tags %}
+          <ul class="ml-auto">
+            {% for tag in post.tags %}
+              <li class="inline-block leading-9">
+                <a class="px-2 py-1 bg-gray-200 rounded" href="/posts?tag={{tag}}">{{tag}}</a>
+              </li>
+            {% endfor %}
+          </ul>
+        {% endif %}
       </div>
       <div class="flex-1 mt-3 markdown">
         {{post.body | default: '' | markdownify}}
       </div>
-      <div class="mt-4 flex items-center justify-between">
-        <div class="flex">
-          <!-- <div class="flex mr-2 text-gray-700 text-sm mr-3">
-               <svg fill="none" viewBox="0 0 24 24"  class="w-4 h-4 mr-1" stroke="currentColor"> <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z"/> </svg>
-               <span>{{post.likes}}</span>
-               </div>
-               <div class="flex mr-2 text-gray-700 text-sm mr-8">
-               <svg fill="none" viewBox="0 0 24 24" class="w-4 h-4 mr-1" stroke="currentColor"> <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M17 8h2a2 2 0 012 2v6a2 2 0 01-2 2h-2v4l-4-4H9a1.994 1.994 0 01-1.414-.586m0 0L11 14h4a2 2 0 002-2V6a2 2 0 00-2-2H5a2 2 0 00-2 2v6a2 2 0 002 2h2v4l.586-.586z"/> </svg>
-               <span>8</span>
-               </div> -->
-        </div>
-        <span>
-          {% if post.id == root_post.id and root_post.creator.id == current_profile.id %}
-            <a href="/dashboard/posts/edit?id={{ post.id }}" class="pr-5">
-              {{ 'app.models.posts.edit' | t }}
-            </a>
-          {% elsif post.creator_id == current_profile.id %}
-            <a href="/dashboard/posts/edit_answer?id={{ post.id }}" class="pr-5">
-              {{ 'app.models.posts.edit_answer' | t }}
-            </a>
-          {% endif %}
-          {% for tag in post.tags %}
-            <a class="px-2 py-1 bg-gray-200 text-blue-400 rounded hover:bg-gray-400" href="/posts?tag={{tag}}">{{tag}}</a>
-          {% endfor %}
-        </span>
+<<<<<<< HEAD
```

Comment:
@z-x You left something after resolving conflict.

Reply:
Done. :)